packages/care/package.py

```diff
@@ -87,6 +93,7 @@ class Care(CachedCMakePackage, CudaPackage, ROCmPackage):
     variant(
         "implicit_conversions",
         default=False,
+        when="@:2024",
```

Comment:
Should this be 0.14.1 (since care doesn't use the same versioning as camp/raja/chai/umpire)?

Reply:
🙈